go.mod

```diff
@@ -6,15 +6,15 @@ require (
 	github.com/cloudflare/cfssl v1.5.0 // for `kubebuilder alpha config-gen`
 	github.com/gobuffalo/flect v0.2.2
 	github.com/joelanford/go-apidiff v0.1.0
-	github.com/onsi/ginkgo v1.15.0
-	github.com/onsi/gomega v1.10.5
+	github.com/onsi/ginkgo v1.16.4
+	github.com/onsi/gomega v1.13.0
```

Comment:
Do we need to update that? See : 

github.com/onsi/ginkgo
  Incompatible changes:
  - GINKGO_VERSION: value changed from "1.15.0" to "1.16.4"

That breaks the api-diff

Reply:
However, in this case, I was also in favor of. We might need to check the api-diff as well. c/c @estroz

Reply:
I don't think this api-diff conflict matters in practice.